app/src/main/java/com/example/cardlink/adapter/ContactAdapter.kt

```diff
@@ -49,7 +49,7 @@ class ContactAdapter(
         viewHolder?.name?.text = contact.name
         viewHolder?.occupation?.text = contact.occupation
         viewHolder?.person = contact
-        viewHolder?.photo?.let { downloadUserImage(contact.primaryKey, it) }
```

Comment:
Does this mean we don't see the user's profile image in the network page anymore? I'm guessing you commented this out while testing?

Reply:
there is a delay when network listview is first populated and some instances where there were duplicate images. will look further into it.